<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>TableView</string>
     </property>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>PlainTextEdit</string>
     </property>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>371</width>
        <height>501</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actFileOpen"/>
   <addaction name="actFileSaveAs"/>
   <addaction name="actModelDataPreview"/>
   <addaction name="actAddLine"/>
   <addaction name="actInsLine"/>
   <addaction name="actDelLine"/>
   <addaction name="actLeft"/>
   <addaction name="actRight"/>
   <addaction name="actCenter"/>
   <addaction name="actBold"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actFileOpen">
   <property name="text">
    <string>打开文件</string>
   </property>
  </action>
  <action name="actFileSaveAs">
   <property name="text">
    <string>另存文件</string>
   </property>
  </action>
  <action name="actModelDataPreview">
   <property name="text">
    <string>模型数据预览</string>
   </property>
  </action>
  <action name="actAddLine">
   <property name="text">
    <string>加入行</string>
   </property>
  </action>
  <action name="actInsLine">
   <property name="text">
    <string>插入行</string>
   </property>
  </action>
  <action name="actDelLine">
   <property name="text">
    <string>删除行</string>
   </property>
  </action>
  <action name="actLeft">
   <property name="text">
    <string>居左</string>
   </property>
  </action>
  <action name="actRight">
   <property name="text">
    <string>居右</string>
   </property>
  </action>
  <action name="actCenter">
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
